distribution/lib/Standard/Visualization/0.0.0-dev/src/Scatter_Plot.enso

```diff
@@ -175,18 +175,17 @@ limit_data limit data = case limit of
             extreme + data.take (Index_Sub_Range.Sample (limit - extreme.length))
 
 ## PRIVATE
-json_from_table : Table -> [Int]|Nothing -> Int|Nothing -> Json
+json_from_table : Table -> [Int]|Nothing -> Int|Nothing -> JS_Object
 json_from_table table bounds limit =
     data = table.point_data |> bound_data bounds |> limit_data limit
     axes = table.axes
-    Json.from_pairs <| [[data_field,data], [axis_field, axes]]
+    '{"' + data_field + '": [' + (data.join ",") + '],' + '"' + axis_field + '":' + axes.to_json + '}'
 
 ## PRIVATE
-json_from_vector : Vector Any -> [Int]|Nothing -> Int|Nothing -> Json
+json_from_vector : Vector Any -> [Int]|Nothing -> Int|Nothing -> JS_Object
 json_from_vector vec bounds limit =
     data = vec.point_data |> bound_data bounds |> limit_data limit
-    axes = [axis_field, Nothing]
-    Json.from_pairs <| [[data_field, data], axes]
+    '{"' + data_field + '": [' + (data.join ",") + '],' + '"' + axis_field + '":null}'
```

Comment:
Something seems not right here.

The return type is `JS_Object` but actually we return a Text JSON representation. That is not consistent. I'm not sure what was the intent here, but we need to make this consistent.

Also why create the JSON by hand? This looks a bit ugly, the old `Json.from_pairs` looked _much_ cleaner IMO. Can we go back to sth like `JS_Object.from_pairs`? If we need the output as Text, we can still create the JS_Object and call `to_json` on it.

I don't think this method is a right place to rely on 'details' like how to put a `null` field in JSON format - we have this handled with `JSON.stringify` and let's rely on it and use our higher-level tools like `JS_Object.from_pairs`.